src/EventSourcingDb.Tests/WriteEventsTests.cs

```diff
@@ -168,5 +169,24 @@ public async Task DeserializesEventDataCorrectly()
         Assert.Equal(eventCandidate.Data, data);
     }
 
+    [Fact]
+    public async Task PassesErrorMessageInException()
+    {
+        var client = _container!.GetClient();
+
+        var eventCandidate = new EventCandidate(
+            Source: "https://www.eventsourcingdb.io",
+            Subject: "test", // Invalid subject, should start with a slash
```

Comment:
Instead of adding a comment here, what about using a variable which is appropriately named (maybe somethin such as `invalidSubject` in the simplest case)?

Reply:
Makes good sense. I personally find it helpful to get the context _why_ the subject is invalid so I suggest to leave the comment behind the line with the new variable. But happy to apply what you suggest.